plugin/evm/atomic/state/atomic_trie.go

```diff
@@ -140,7 +144,7 @@ func newAtomicTrie(
 	}
 	// initialize to EmptyRootHash if there is no committed root.
 	if root == (common.Hash{}) {
-		root = types.EmptyRootHash
+		root = EmptyAtomicRootHash
```

Comment:
optional: we can import this from go-ethereum's upstream type if we don't want to duplicate this

Reply:
I reverted it since it should be good after libevm transition